test/DynamoCoreTests/DummyNodeTests.cs

```diff
@@ -24,6 +25,14 @@ protected override void GetLibrariesToPreload(List<string> libraries)
             base.GetLibrariesToPreload(libraries);
         }
 
+        private void LoadPackage(string packageDirectory)
```

Comment:
I think we can move this function to the base class as it is used in more than on derived class, this one and `PackageDependencyTests`.

Reply:
Yes, moved it DynamoModelTestBase class.